packages/spaces/src/modals/DisclaimerModal.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import get from 'lodash/get';
+import { useEffectAsync } from '@availity/hooks';
+import { avWebQLApi } from '@availity/api-axios';
+import ReactMarkdown from 'react-markdown';
+import { ModalBody } from 'reactstrap';
+import Loader from '../Loader';
+
+const disclaimerQuery = `query configurationFindOne($id: ID!) {
+  configurationFindOne(id: $id) {
+    ... on Text {
+      description
+    }
+  }
+}`;
+
+const DisclaimerModal = ({ disclaimerId }) => {
+  const [disclaimer, setDisclaimer] = useState('');
+
+  const fetchDisclaimer = async () => {
```

Comment:
I think we should change the `useEffectAsync` to `useEffect` and move this function inside of there